src/test-data/ModificationsData.js

```diff
@@ -0,0 +1,62 @@
+const props = [
+  {
+    name: "Name",
```

Comment:
Can we make these values something like `Some Prop Name` and `Some Prop Value` so it's a bit more obvious in the rending of the component?